R/R.r

```diff
@@ -50,10 +51,24 @@ r_env_vars <- function() {
     "TAR" = auto_tar())
 
   if(is.na(Sys.getenv("NOT_CRAN", unset = NA))) {
-    c(vars, "NOT_CRAN" = "true")
-  } else {
-    vars
+    vars <- c(vars, "NOT_CRAN" = "true")
   }
+
+  vars
+}
+
+# Create a temporary .Rprofile based on the current "repos" option
+# and return a named vector that corresponds to environment variables
+# that need to be set to use this .Rprofile
+r_create_repos_profile <- function() {
```

Comment:
Maybe this would be better called `r_profile()`?  We might expand it to include other options in the future.